src/kanata/mod.rs

```diff
@@ -478,6 +513,20 @@ impl Kanata {
             tray_icon: None,
             #[cfg(all(target_os = "windows", feature = "gui"))]
             icon_match_layer_name: cfg.options.icon_match_layer_name,
+            #[cfg(all(target_os = "windows", feature = "gui"))]
```

Comment:
With the number of options all conditionally compiled with gui growing, it seems like it's time to group these under a nested struct.